src/Toolkit/doc/index.rst

```diff
@@ -49,7 +49,7 @@ For example, if you want to install a `Button` component, you will find the foll
 
 .. code-block:: terminal
 
-    $ php bin/console ux:toolkit:install-component Button --kit=<kitName>
+    $ php bin/console ux:install Button --kit=<kitName>
```

Comment:
The command `ux:toolkit:install-component` has been renamed to `ux:install`, cc @smnandre